### PR TITLE
test cases and additional documentation updates

### PR DESCRIPTION
**Pull request by maheshc01**
updated to include positive test cases as per CAMARA guidelines.
Also made few updates to documentation.

### CODE REVIEW

**Comment by maheshc01** [comment]
@Kevsy fro review and merge.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+* for the first alpha or release-candidate API version, all changes since the release of the previous public API version
+* for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
+* for a public API version, the consolidated changes since the release of the previous public API version
+
+
+## Release Notes
+
+This release contains the definition and documentation of
+* Connecitvity Insights API
```
Comment (Kevsy) [inline CHANGELOG.md:21]:
```suggestion
* Connectivity Insights API
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Changelog Connectivity Insights
+
+NOTE:
+
+## Table of contents
+
+- [r1.1]
```
Comment (Kevsy) [inline CHANGELOG.md:7]:
```suggestion
- [r1.1 -  rc]
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Release Notes
+
+This release contains the definition and documentation of
+* Connecitvity Insights API
+* Application Profiles API
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
```
Comment (Kevsy) [inline CHANGELOG.md:26]:
```suggestion
* Identity and Consent Management v0.2.0-rc.1
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+## Connectivity Insights API v0.4.0-rc.1
+
+** 0.4.0-rc.1 is the first release-candidate version for connectivity insights**
+
+
+### Main changes
+- This version is primarily to align with the changes to Commonalities 0.4-rc.1 and Identity and Consent Management v0.2.0-rc.1.
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc]
```
Comment (Kevsy) [inline CHANGELOG.md:38]:
```suggestion
  - View it on ReDoc:
    - [connectivity-insights.yaml](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.1/code/API_definitions/connectivity-insights.yaml&nocors)
    
    - [application-profiles.yaml](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.1/code/API_definitions/application-profiles.yaml&nocors)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  - [View it on Swagger Editor]
```
Comment (Kevsy) [inline CHANGELOG.md:39]:
```suggestion
  - [View it on Swagger Editor]
    - [connectivity-insights.yaml]((https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.1/code/API_definitions/connectivity-insights.yaml))
      - [application-profiles.yaml]((https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.1/code/API_definitions/application-profiles.yaml))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  - OpenAPI [YAML spec file]
```
Comment (Kevsy) [inline CHANGELOG.md:40]:
```suggestion
  - OpenAPI
    -  [connectivity-insights.yaml](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.1/code/API_definitions/connectivity-insights.yaml)
    -  [application-profiles.yaml](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.1/code/API_definitions/application-profiles.yaml)
```

File: code/API_definitions/application-profiles.yaml [hunk 7]
```diff
@@ -2,13 +2,15 @@ openapi: 3.0.3
...
+  contact:
+    email: sp-coi@lists.camaraproject.org
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
   description: |
     Application profiles allows application developers to share all the
-    information about their application which would be relavant in
-    network/CAMARA APIs related decision making.  //todo
+    information about their application which would be relavant for network/
```
Comment (Kevsy) [inline code/API_definitions/application-profiles.yaml:12]:
```suggestion
    information about their application which would be relevant for network/
```

File: documentation/API_documentation/ConnecitvityInsights-SequenceDiagram.png
Comment (Kevsy) [inline documentation/API_documentation/ConnecitvityInsights-SequenceDiagram.png:1]: The filename has a typo (Connecitvity -> Connectivity)

**Review by Kevsy — CHANGES_REQUESTED** [review]
Looks good other than the comments provided (with suggested fixes)

**Review by Kevsy — APPROVED** [review]

**Pull request merged**